docs/en/docs/insight/user-guide/collection-manag/probe-module.md

```diff
@@ -1,16 +1,30 @@
+---
+MTPE: WANG0608GitHub
+Date: 2024-09-23
+---
+
 # Custom Probe Methods
 
+Insight uses the Blackbox Exporter provided by Prometheus as a blackbox monitoring solution, allowing detection of target instances via HTTP, HTTPS, DNS, ICMP, TCP, and gRPC. It can be used in the following scenarios:
+
+- HTTP/HTTPS: URL/API availability monitoring
+- ICMP: Host availability monitoring
+- TCP: Port availability monitoring
+- DNS: Domain name resolution
+
 In this page, we will explain how to configure custom probe methods in an existing Blackbox ConfigMap.
-We will use the HTTP probe method as an example to demonstrate how to modify the ConfigMap to achieve custom HTTP probing.
+
+ICMP probing is not enabled by default in Insight because it requires higher permissions. Therfore We will use the HTTP probe method as an example to demonstrate how to modify the ConfigMap to achieve custom HTTP probing.
```

Comment:
```suggestion
ICMP probing is not enabled by default in Insight because it requires higher permissions. Therfore We will use the HTTP probe as an example to demonstrate how to modify the ConfigMap to achieve custom HTTP probing.
```